addons/markers/XEH_postInit.sqf

```diff
@@ -18,3 +18,24 @@ GVAR(currentMarkerPosition) = [];
 GVAR(currentMarkerAngle) = 0;
 GVAR(currentMarkerColorConfigName) = "";
 GVAR(currentMarkerConfigName) = "";
+
+// set marker pos local on every computer (prevent markers visible for everyone)
+[QGVAR(applyMarkerPosLocal), {
+    params["_marker", "_pos"];
+    _marker setMarkerPosLocal _pos;
+}] call CBA_fnc_addEventHandler;
+
+["ace_settingsInitialized", {
+    if (GVAR(movableMarkersEnabled)) then {
+        addMissionEventHandler ["Map", {
+            params ["_isOpen"];
+
+            if (_isOpen) then {
+                (findDisplay 12 displayCtrl 51) ctrlAddEventHandler ["MouseButtonDown", FUNC(onMouseButtonDownMap)];
```

Comment:
We usually only add the EH once, see https://github.com/acemod/ACE3/blob/master/addons/map/XEH_postInitClient.sqf#L24

This causes problems with the marker alpha because it saves the 0.5, also sends multiple events over the network

```
[{
    !isNull (findDisplay 12)
}, {
    (findDisplay 12 displayCtrl 51) ctrlAddEventHandler ["MouseButtonDown", FUNC(onMouseButtonDownMap)];
    (findDisplay 12 displayCtrl 51) ctrlAddEventHandler ["MouseButtonUp", FUNC(onMouseButtonUpMap)];
}] call CBA_fnc_waitUntilAndExecute;
```
Should work nicely